R/BASiCS_DenoisedCounts.R

```diff
@@ -36,7 +40,7 @@
 #'
 #' @rdname BASiCS_DenoisedCounts
 #' @export
-BASiCS_DenoisedCounts <- function(Data, Chain) {
+BASiCS_DenoisedCounts <- function(Data, Chain, IncludeSpikes = TRUE) {
```

Comment:
To be consistent with `BASiCS_MCMC` maybe we should use `WithSpikes` instead of `IncludeSpikes`?